controllers/statusController.js

```diff
@@ -0,0 +1,57 @@
+const conferences = require('../lib/conferences')
+const config = require("../config")
+
+/*
+GET /status
+code 200 -> all is well
+code 500 -> error
+*/
+module.exports.getStatus = async (req, res) => {
+  const status = {
+    USE_OVH_ROOM_API : config.USE_OVH_ROOM_API,
+    FEATURE_DISPLAY_STATS_ON_LANDING : config.FEATURE_DISPLAY_STATS_ON_LANDING,
+    FEATURE_RESERVATIONS : config.FEATURE_RESERVATIONS,
+    FEATURE_STATS_PAGE : config.FEATURE_STATS_PAGE,
+    version: require('../package').version,
+  }
+
+  try {
+    if (config.USE_OVH_ROOM_API) {
+      await conferences.getRoomsStats()
```

Comment:
Tu ne te sers pas du retour de ces fonctions ? 🤔

Reply:
non, si l'API OVH ne retourne pas 200, getRoomsStats va throw. Donc juste le fait que la fonction ne throw pas, ca suffit à dire que la requete a réussi et que OVH is up.